### PR TITLE
<ColorPicker/> - last tab in the color picker doesn't stretch to the end

### PR DESCRIPTION
**Pull request by Faradey27**
### What changed

<ColorPicker/> - last tab in the color picker doesn't stretch to the end
Added new minWidth property for Tabs to be able to override hardcoded css minWidth property

### Why it changed

to fix layout problems

---

- [x] Change is tested
- [x] Change is documented
- [x] Build is green
- [x] Change of UX is approved by [wuwa](https://github.com/wuwa) or [milkyfruit](https://github.com/milkyfruit)

### CODE REVIEW

**Comment by Faradey27** [comment]
@shlomitc

**Comment by hepiyellow** [comment]
screen shot 2018-06-18 at 14 13 29

Doesn't seem to be fixed

File: src/Tabs/Tabs.js [hunk 1]
```diff
@@ -34,14 +34,15 @@ class Tabs extends WixComponent {
...
+  hasDivider: PropTypes.bool,
+  items: TabPropTypes.items.isRequired,
+  minWidth: TabPropTypes.width,
   type: TabPropTypes.type,
-  width: TabPropTypes.width,
   sideContent: TabPropTypes.sideContent,
-  dataHook: PropTypes.string,
-  hasDivider: PropTypes.bool
+  width: TabPropTypes.width,
+  onClick: TabPropTypes.onClick
```
Comment (hepiyellow) [inline src/Tabs/Tabs.js]: If you reorder the prop types alphabetically, then onClick should be before width

File: src/Tabs/core/TabItems/index.js [hunk 2]
```diff
@@ -31,7 +31,11 @@ class TabItems extends React.Component {
     const className = classNames(styles.itemsContainer, styles[type]);
 
     return (
-      <ul className={className} data-hook={dataHook}>
+      <ul
+        className={className}
+        data-hook={dataHook}
+        style={{minWidth: this.props.minWidth}}
```
Comment (hepiyellow) [inline src/Tabs/core/TabItems/index.js]: Are you sure you can pass a number here?

Comment (hepiyellow) [reply src/Tabs/core/TabItems/index.js]: It is getting `0px` now

File: src/ColorPicker/color-picker-converter.js [hunk 3]
```diff
@@ -50,7 +50,12 @@ export default class ColorPickerConverter extends WixComponent {
 
     return (
       <div>
-        <Tabs items={tabs} activeId={activeTab} onClick={this.changeTab}/>
+        <Tabs
+          minWidth={0}
+          items={tabs}
+          activeId={activeTab}
+          onClick={this.changeTab}
```
Comment (hepiyellow) [inline src/ColorPicker/color-picker-converter.js]: I think you should pass here `type={'uniformFull'}`

File: src/Tabs/core/TabItems/index.js (hunk 2 above)
Comment (Faradey27) [reply src/Tabs/core/TabItems/index.js]: @hepiyellow yes, i am sure, if minWidth === undefined, then minWidth in style will be ignored

how did you get 0px?

File: src/ColorPicker/color-picker-converter.js (hunk 3 above)
Comment (Faradey27) [reply src/ColorPicker/color-picker-converter.js]: no, to make it workable with uniformFull it require to change internals of tabs

with uniformFull - http://prntscr.com/jwaf5e

@hepiyellow

**Comment by Faradey27** [comment]
@hepiyellow build is green again

**Pull request merged**